src/Microsoft.PowerShell.ConsoleHost/host/msh/CommandLineParameterParser.cs

```diff
@@ -320,6 +321,23 @@ internal Collection<CommandParameter> Args
             }
         }
 
+        internal string? ConfigurationFile
```

Comment:
could you be sure to add telemetry for this new parameter? you may need to rebase.